packages/utils/odsp-doclib-utils/package.json

```diff
@@ -25,7 +55,7 @@
 	},
 	"main": "dist/index.js",
 	"module": "lib/index.mjs",
-	"types": "dist/index.d.ts",
+	"types": "./dist/odsp-doclib-utils-public.d.ts",
```

Comment:
I'm guessing non-Node16 clients use the 'types' export?

Updated to point at public so that older module resolution schemes are not a backdoor to internals.

Reply:
Yeah keeping the types field as a fallback is good practice AFAIK. I usually see them without the leading `./` but I don't know if it matters. There's a policy that's currently disabled that will validate the types field matches the '.' exports types.